### PR TITLE
Update next build docs to include the --profile argument

### LINKED ISSUE #15720: [Docs] Update next build docs to include the --profile argument
`--profile` enables production profiling for React and was released in [Next.js 9.5](https://nextjs.org/blog/next-9-5#production-react-profiling)

### PR DESCRIPTION
**Pull request by agarwalrounak**
Closes #15720

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12s | -224ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.167 | 2.174 | ⚠️ +0.01 |
| / avg req/sec | 1153.71 | 1149.97 | ⚠️ -3.74 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.388 | 1.27 | -0.12 |
| /error-in-render avg req/sec | 1800.57 | 1968.14 | +167.57 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..9e81.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8dccf8c..edef.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-85e7a5d..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| index.html gzip | 944 B | 944 B | ✓ |
| link.html gzip | 951 B | 951 B | ✓ |
| withRouter.html gzip | 939 B | 939 B | ✓ |
| Overall change | 2.83 kB | 2.83 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 12.6s | 15.4s | ⚠️ +2.9s |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..9e81.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8dccf8c..edef.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-85e7a5d..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: 13883cd93d0f08393ba1c7644d81594b34d3989b

File: docs/api-reference/cli.md [hunk 1]
```diff
@@ -48,6 +48,14 @@ NODE_OPTIONS='--inspect' next
 
 The first load is colored green, yellow, or red. Aim for green for performant applications.
 
+With Next.js 9.5, you can now **enable production profiling for React** with the `--profile` flag in `next build`:
```
Comment (timneutkens) [inline docs/api-reference/cli.md:51]:
```suggestion
You can enable production profiling for React with the `--profile` flag in `next build`. This requires Next.js 9.5:
```

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 11.3s | 11.5s | ⚠️ +213ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.202 | 2.099 | -0.1 |
| / avg req/sec | 1135.49 | 1190.82 | +55.33 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.351 | 1.293 | -0.06 |
| /error-in-render avg req/sec | 1850.94 | 1932.89 | +81.95 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..9e81.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8dccf8c..edef.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-85e7a5d..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| index.html gzip | 944 B | 944 B | ✓ |
| link.html gzip | 951 B | 951 B | ✓ |
| withRouter.html gzip | 939 B | 939 B | ✓ |
| Overall change | 2.83 kB | 2.83 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.2s | -444ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..9e81.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-8dccf8c..edef.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-85e7a5d..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: f5fd3682d0d0743adfed09e2201ad1bf5bb34e6e

**Review by Timer — APPROVED** [review]
LGTM, thanks!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 14.3s | 14.3s | -67ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.613 | 2.627 | ⚠️ +0.01 |
| / avg req/sec | 956.69 | 951.58 | ⚠️ -5.11 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.546 | 1.563 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1617.34 | 1599.37 | ⚠️ -17.97 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..afbf.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-76d73eb..95d8.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.11 kB | 6.11 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-0b6ca7d..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-75..1629.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-e97733c..af21.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-fbb1d2e..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 272 B | 272 B | ✓ |
| _buildManife..dule.js gzip | 280 B | 280 B | ✓ |
| Overall change | 552 B | 552 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| index.html gzip | 945 B | 945 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 939 B | 939 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| buildDuration | 15.4s | 16.1s | ⚠️ +733ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..afbf.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-76d73eb..95d8.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.11 kB | 6.11 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-0b6ca7d..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| polyfills-75..1629.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-e97733c..af21.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-fbb1d2e..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _buildManifest.js gzip | 272 B | 272 B | ✓ |
| _buildManife..dule.js gzip | 280 B | 280 B | ✓ |
| Overall change | 552 B | 552 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | agarwalrounak/next.js update-docs | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: 999073878001c6f85bff061a4c2aa6bf5c989049

**Pull request merged**